client/src/explorer_common/explorer_components.js

```diff
@@ -99,7 +99,7 @@ const get_children_content = ({
           typeName="ul"
           className="ExplorerNodeContainer__ChildrenList"
           staggerDurationBy="0"
-          duration={500}
+          duration={400}
```

Comment:
Do we still need this change?

If i'm correct, that duration is for the `FlipMove`

Reply:
I forgot to note this in the PR, but I added this change on purpose. I just thought `500` was a bit too slow. 
More of a preference-based change than anything. Open to change.